package.json

```diff
@@ -6,8 +6,8 @@
     "ng": "nx",
     "postinstall": "node ./decorate-angular-cli.js && ngcc --properties es2015 browser module main",
     "start": "nx serve",
-    "start:api": "nx serve snapodds-api",
-    "build": "nx run-many --target=build --projects=snapodds-builder,snapodds",
+    "start:demo": "nx build snapodds-demo && nx serve snapodds-demo",
```

Comment:
nx serve will automatically trigger the build -> see project.json
`"serve": {
      "executor": "@nrwl/node:execute",
      "options": {
        "buildTarget": "snapodds-demo:build"
      }
    },
 "`